crates/yakui-vulkan/examples/demo.rs

```diff
@@ -277,12 +277,23 @@ impl VulkanTest {
             .engine_version(0)
             .api_version(vk::make_api_version(0, 1, 3, 0));
 
-        let extension_names =
+        #[allow(unused_mut)]
+        let mut extension_names =
             ash_window::enumerate_required_extensions(window.display_handle().unwrap().as_raw())
                 .unwrap()
                 .to_vec();
 
+        #[cfg(target_os = "macos")]
+        extension_names.push(ash::khr::portability_enumeration::NAME.as_ptr());
+
+        let create_flags = if cfg!(any(target_os = "macos", target_os = "ios")) {
```

Comment:
What's the impact of checking for macOS and iOS versus just iOS? Do they support different extensions?

Reply:
Just a bad copypasta, sorry. Since we don't support iOS anyway I'll clean this up to just be macOS.